server/request_object.cc

```diff
@@ -96,6 +97,36 @@ void RequestObject::listen_for_incoming_data(std::function<void()> callback,
   resume();  // resume reading if it was paused
 }
 
+static void generate_uuid(const std::string& raw_str, char uuid[16]) {
```

Comment:
can we use existing uuid.get_string_uuid() and trim if required

Reply:
No. Another format